assets/express/src/api/index.ts

```diff
@@ -1,6 +1,6 @@
-import { DbCollection } from 'src/database/intefaces';
+import { DbCollection } from 'src/database/interfaces';
```

Comment:
IMO here we should import with `@database`

Reply:
This PR doesn't aim to fix alias usage